lib/github/middleware.js

```diff
@@ -75,6 +69,14 @@ module.exports = function middleware(callback) {
     const gitHubInstallationId = context.payload.installation.id;
     context.log = context.log.child({ gitHubInstallationId });
 
+    // Ignore all incoming webhooks if maintenance mode is enabled
+    if (isMaintenanceMode()) {
+      context.sentry.setTag('maintenanceMode', true);
+      Sentry.captureMessage('Maintenance mode ENABLED - Ignoring event');
+      context.log({ maintenanceMode: true }, 'Maintenance mode ENABLED - Ignoring event');
```

Comment:
How do we get the installation ID out of this log event? What's `context.log.child({ gitHubInstallationId })` in line 70 doing? Is it adding the installation to the JSON structure of the log event?

It would probably be safer to just include the installation ID in the log message itself to avoid any issues in extracting the log data from Splunk later.

Reply:
that's correct, line 70 is making `context.log` always include this specific installationId with all requests as a piece of filterable data.  It's already in splunk and we can test it after deploying it to staging to by getting all the `installationId` of all messages with `maintenanceMode:true`.

Reply:
Ok, I just tested the export in Splunk and it returns all fields in the CSV export, so we should be good.

Reply:
nice :)